src/integrations/BingAds/browser.js

```diff
@@ -22,31 +24,32 @@ class BingAds {
         (n = d.createElement(t)),
         (n.src = r),
         (n.async = 1),
-        (n.setAttribute("data-loader", LOAD_ORIGIN)),
+        n.setAttribute('data-loader', LOAD_ORIGIN),
         (n.onload = n.onreadystatechange =
           function () {
             let s = this.readyState;
-            (s && s !== "loaded" && s !== "complete") ||
+            (s && s !== 'loaded' && s !== 'complete') ||
               (f(), (n.onload = n.onreadystatechange = null));
           }),
         (i = d.getElementsByTagName(t)[0]),
         i.parentNode.insertBefore(n, i);
-    })(window, document, "script", "https://bat.bing.com/bat.js", "uetq");
+    })(window, document, 'script', 'https://bat.bing.com/bat.js', this.uniqueId);
```

Comment:
Can we add the documentation for this? I am not able to find [here](https://help.ads.microsoft.com/#apex/ads/en/56686/2).

Reply:
Updated the links in the PR desc.